src/modules/components/proposal/proposalActions/proposalActionsTypes/proposalActionUpdateMetadata.ts

```diff
@@ -0,0 +1,51 @@
+import { type IProposalAction } from './proposalAction';
+import { type ProposalActionType } from './proposalActionType';
+
+export interface IMetadataLink {
+    /**
+     * Human readable label for link.
+     */
+    label: string;
+    /**
+     * link href.
+     */
+    href: string;
+    /**
+     * link target.
+     */
+    target: string;
```

Comment:
We won't get a `target` attribute from the DAO metadata, we can just set it to `_blank` on the custom action UI as these will be external urls